framework/move-stdlib/sources/configs/features.move

```diff
@@ -209,6 +209,14 @@ module std::features {
         is_enabled(DIEM_UNIQUE_IDENTIFIERS)
     }
 
+    /// Whether the new epoch trigger logic is enabled.
+    /// Lifetime: transient
+    const EPOCH_TRIGGER_ENABLED: u64 = 24;
```

Comment:
The naming and usage indicates packing both a boolean flag, as well as functional value (24) into a singular variable. I think it would be cleaner as two values, however, this is beyond the scope of this pr and perhaps something to add to the todo list.

Reply:
Yes, it's not idea.. The convention of the features (which comes from vendor code) is to give them IDs. The assumption is that the flag is always a 1 bit.